frontend/src/components/Pages/MissionDefinitionPage/MissionDefinitionPage.tsx

```diff
@@ -94,45 +123,83 @@ function MissionDefinitionPageBody({ missionDefinition, updateMissionDefinition
         return TranslateText('Inspection required every') + ' ' + returnStringArray.join(', ')
     }
 
+    const onEdit = (editType: string) => {
+        return () => {
+            setIsEditDialogOpen(true)
+            setSelectedField(editType)
+        }
+    }
+
     return (
         <>
-            <KeyValuePairDisplay left={TranslateText('Comment')} right={missionDefinition.comment} />
-            <KeyValuePairDisplay
-                left={TranslateText('Inspection frequency')}
-                right={displayInspectionFrequency(missionDefinition.inspectionFrequency)}
-            />
-            <KeyValuePairDisplay left={TranslateText('Area')} right={missionDefinition.area.areaName} />
-            <KeyValuePairDisplay left={TranslateText('Deck')} right={missionDefinition.area.deckName} />
-            <KeyValuePairDisplay left={TranslateText('Plant')} right={missionDefinition.area.plantCode} />
-            <KeyValuePairDisplay left={TranslateText('Installation')} right={missionDefinition.area.installationCode} />
-            <KeyValuePairDisplay
-                left={TranslateText('Mission source')}
-                right={TranslateText(missionDefinition.sourceType)}
-            />
-            <Button
+            <StyledFormContainer>
+                <KeyValuePairDisplay
+                    left={TranslateText('Name')}
+                    right={missionDefinition.name}
+                    onEdit={onEdit('name')}
+                />
+                <KeyValuePairDisplay left={TranslateText('Area')} right={missionDefinition.area.areaName} />
+                <KeyValuePairDisplay left={TranslateText('Deck')} right={missionDefinition.area.deckName} />
+                <KeyValuePairDisplay left={TranslateText('Plant')} right={missionDefinition.area.plantCode} />
+                <KeyValuePairDisplay
+                    left={TranslateText('Installation')}
+                    right={missionDefinition.area.installationCode}
+                />
+                <KeyValuePairDisplay
+                    left={TranslateText('Mission source')}
+                    right={TranslateText(missionDefinition.sourceType)}
+                />
+                <KeyValuePairDisplay
+                    left={TranslateText('Comment')}
+                    right={missionDefinition.comment}
+                    onEdit={onEdit('comment')}
+                />
+                <KeyValuePairDisplay
+                    left={TranslateText('Inspection frequency')}
+                    right={displayInspectionFrequency(missionDefinition.inspectionFrequency)}
+                    onEdit={onEdit('inspectionFrequency')}
+                />
```

Comment:
If we write something that isn't a number "NaN" will show. Suggest emptying such that the prompt stays empty and add a "invalid input" feedback to the user

Reply:
![image](https://github.com/equinor/flotilla/assets/33518988/cfe2e783-2c9b-485a-bbb5-abe86e55d52b)

Reply:
I'll just not allow non-numeric input